packages/usa-form-group/src/styles/_usa-form-group.scss

```diff
@@ -11,11 +11,13 @@
 
 // Block input elements
 .usa-form-group--error {
-  @include u-border-left(0.5, "error-dark");
-  padding-left: units(2);
   position: relative;
 
-  @include at-media("desktop") {
-    margin-left: units(-2.5);
+  &::before {
+    @include u-border-left(0.5, "error-dark");
+    content: "";
+    height: 100%;
+    left: units("neg-105");
```

Comment:
**question:** This brings the border much closer to the content, was that intentional?

Reply:
Yeah, removing the `units(-2.5)` of margin prevents the border from being pushed out of the parent div. This is what was causing the border to disappear from our preview in desktop widths.